readme.md

```diff
@@ -104,65 +102,67 @@ app.listen(4000, () => {
 });
 ```
 
-We've required the Express module. We then create another variable that invokes
-the module. The app variable is where we can set up which port to listen to and
-the different requests the server should listen for.
+We've required the Express module which is a function that returns an instance of a web app.
+We invokes the  module instantiating a constant `app` which holds all the methods and state we use to write and run our web app.
```

Comment:
Singular `invoke` instead of plural, also there's an extra space before `module instantiating a...`